module/VuFind/src/VuFind/Db/Entity/UserEntityInterface.php

```diff
@@ -384,4 +384,5 @@ public function setEmailVerified(?DateTime $dateTime): static;
      * @return ?DateTime
      */
     public function getEmailVerified(): ?DateTime;
+
```

Comment:
I don't think this blank line is needed -- a `composer fix` may get rid of it automatically and fix the broken build here.